docs/concepts/registries.md

```diff
@@ -43,6 +43,15 @@ public void registerBlocks(RegistryEvent.Register<Block> event) {
 }
 ```
 
+### Registries that aren't Forge Registries
+
+Due to some peculiarities of vanilla code, not all registries are wrapped by Forge. These can be static registries, like `RecipeType`, which are safe to use. There are also dynamic registries, like `ConfiguredFeature` and some other worldgen registries, which are typically represented in JSON. These objects should only be registered this way if there is another registry object that requires it. 
+
+Without the benefit of registry objects, we have to be careful of when we do the registration and how.
+
+Outside your main mod class, have a static field store a `Lazy` of a call to `Registry#register` containing the object you want to register. Later, in FMLCommonSetupEvent, enqueue a call to `Supplier#get` on the object, thus populating it.
```

Comment:
What is a `Lazy`? Why must the field be within a `Lazy`? What does `Registry#register` take in ? Why must this be done in the common setup event? How did `Lazy` become a `Supplier#get`?

Telling someone to use a particular piece of code without some explanation as to reasonings behind it make it difficult for people to understand the specifications and how to adapt them.